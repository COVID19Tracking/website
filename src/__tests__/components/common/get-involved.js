import React from 'react'
import renderer from 'react-test-renderer'
import GetInvolved from '../../../components/common/get-involved'

describe('Components : Common: Get Involved', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <GetInvolved items={([<p>Sample text</p>], [<p>More text</p>])} />,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
