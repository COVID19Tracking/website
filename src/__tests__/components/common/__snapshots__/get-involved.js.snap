// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Common: Get Involved renders correctly 1`] = `
<div>
  <div
    className="getInvolved"
  >
    <div
      className="getInvolvedIcon"
    >
      â†’
    </div>
    <p>
      More text
    </p>
  </div>
</div>
`;
